algotaurus.py

```diff
@@ -423,15 +425,16 @@ def __init__(self, size=15, lines=30):
         self.tkMessageBox = tkMessageBox
 
         # Initial parameters
-        self.size = size
         self.lines = lines
-        self.x = 27
-        self.y = 27
+        self.code = code
```

Comment:
Program will be able to store the code from the coder and transfer it a the new mainloop.